utils/getScenarioContent.tsx

```diff
@@ -7,51 +7,186 @@
 import getClaimStatus from './getClaimStatus'
+import { isDatePast, isValidDate, parseConvertDate } from './formatDate'
+import { isFirstTimeSlotEarlier } from './timeSlot'
 
 export enum ScenarioType {
   Scenario1,
+  Scenario2,
+  Scenario3,
   Scenario4,
   Scenario5,
   Scenario6,
 }
 
 export const ScenarioTypeNames = {
-  [ScenarioType.Scenario1]: 'Pending determination scenario',
+  [ScenarioType.Scenario1]: 'Determination interview: not yet scheduled',
+  [ScenarioType.Scenario2]: 'Determination interview: scheduled',
+  [ScenarioType.Scenario3]: 'Determination interview: awaiting decision',
   [ScenarioType.Scenario4]: 'Generic pending state: pending weeks',
   [ScenarioType.Scenario5]: 'Base state: no pending weeks, no weeks to certify',
   [ScenarioType.Scenario6]: 'Base state: no pending weeks, weeks to certify',
 }
 
+interface PendingDeterminationScenario {
+  scenarioType: ScenarioType
+  pendingDetermination?: PendingDetermination
+}
+
+export const NonPendingDeterminationValues = ['Canceled', 'Complete', 'TRAN', 'INVL', 'IDNC', '1277', 'OTHR', 'ClmCX']
+
+/**
+ * Determine whether the Determination Status is pending.
+ *
+ * Determination Status is considered pending if:
+ * either DeterminationStatus is blank (NULL)
+ * OR DeterminationStatus is neither "Canceled" nor "Complete" nor "TRAN" nor
+ *   "INVL" nor "IDNC" nor "1277" nor "OTHR" nor "ClmCX"
+ */
+export function isDeterminationStatusPending(pendingDetermination: PendingDetermination): boolean {
+  return (
+    !pendingDetermination.determinationStatus ||
+    !NonPendingDeterminationValues.includes(pendingDetermination.determinationStatus)
+  )
+}
+
+/**
+ * Identify whether the first pendingDetermination object is scheduled before the second object.
+ *
+ * If both arguments are scheduled at the same time, this will return false.
+ * Assumes that dates have already been checked for validity.
+ */
+export function isScheduledStrictlyBefore(first: PendingDetermination, second: PendingDetermination): boolean {
+  const firstScheduleDate = parseConvertDate(first.scheduleDate)
+  const secondScheduleDate = parseConvertDate(second.scheduleDate)
+
+  // If the first appointment is scheduled before the second.
+  if (firstScheduleDate < secondScheduleDate) {
+    return true
+  }
+  // If the first appointment is scheduled after the second.
+  else if (firstScheduleDate > secondScheduleDate) {
+    return false
+  }
+  // If both appointments are on the same date...
+  else {
+    // ...then compare the time slots.
+    const isEarlier = isFirstTimeSlotEarlier(first.timeSlotDesc, second.timeSlotDesc)
+
+    // It's possible for both time slots to be improperly formatted, in which case it doesn't
+    // matter which appointment is said to be first, since they are on the same date.
+    if (!isEarlier) {
+      return false
+    }
+    // Otherwise, return the appointment with the earlier time slot start time.
+    else {
+      return isEarlier
+    }
```

Comment:
I think you can return `!!isEarlier` - that will return false for `false` or `null`, `true` for `true`

Reply:
Neat! I don't think I've seen that syntax before.